docs/UsersGuide/source/Introduction.rst

```diff
@@ -1,177 +1,330 @@
-and run once the required bundled libraries (NCEPLIBS) and third-party libraries (NCEPLIBS-external)
-are built.
-
-Limited-Test (level 3) and Build-Only (level 4) computational platforms are those in which
-the developers have built the code but little or no pre-release testing has been conducted,
-respectively. A complete description of the levels of support, along with a list of preconfigured
-and configurable platforms can be found in the `SRW Application wiki page 
-<https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`_.
-
-User Support, Documentation, and Contributing Development
-=========================================================
-
-A forum-based, online `support system <https://forums.ufscommunity.org>`_ with topical sections
-provides a centralized location for UFS users and developers to post questions and exchange
-information. The forum complements the formal, written documentation, summarized here for ease of
-use.  
+----------------------------
+
+The SRW Application has a portable CMake-based build system that packages together all the components required to build the SRW Application. Once built, users can generate a Rocoto-based workflow that will run each task in the proper sequence (see `Rocoto documentation <https://github.com/christopherwharrop/rocoto/wiki/Documentation>`__ for more on workflow management). Individual components can also be run in a stand-alone, command line fashion. 
+
+The SRW Application allows for configuration of various elements of the workflow. For example, users can modify the parameters of the atmospheric model, such as start and end dates, duration, time step, and the physics suite for the forecast. 
+
+This SRW Application release has been tested on a variety of platforms widely used by researchers, including NOAA High-Performance Computing (HPC) systems (e.g. Hera, Orion), cloud environments, and generic Linux and macOS systems. Four `levels of support <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`_ have been defined for the SRW Application. Preconfigured (Level 1) systems already have the required external libraries (e.g., NCEPLIBS) available in a central location. The SRW Application is expected to build and run out-of-the-box on these systems, and users can :ref:`download the SRW code <DownloadSRWApp>` without first installing prerequisites. On other platforms, the SRW can be :ref:`run within a container <QuickstartC>` that includes the HPC-Stack, or the required libraries will need to be installed as part of the :ref:`SRW build <BuildRunSRW>` process. Once these prerequisite libraries are installed, applications and models should build and run successfully. However, users may need to perform additional troubleshooting on Level 3 or 4 systems since little or no pre-release testing has been conducted on these systems. 
+
```

Comment:
I'm not sure if Linux and macOS will be supported for this release?  I thought there some question on whether that would be possible.  I may just not be in the loop though.  @mkavulich, @mark-a-potts?

Reply:
I was thinking of it as being on a spectrum, with plenty of support for Level 1 systems, and significantly less support for other systems, particularly Level 3/4 (at the moment). But I certainly need to know if that's not the case! ;)

Reply:
That is definitely the case.  [Here](https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers) is how the tiers work.  So with Linux/MacOS being Tier 3, the App at least needs to build on those platforms, but testing whether it runs is limited.  But I don't think the build is working at the moment. Correct, @mkavulich, @mark-a-potts?

Reply:
We have been working to get the build of hpc-stack on macOS documented and that should be complete by the release. If hpc-stack builds, then the SRW should build as well. We also have the SRW building in an ubuntu-20.04 container, so I think we can say that version of Linux is supported. One other note, I would change "NCEPLIBS" to "hpc-stack' in section 89 above.

Reply:
I suppose it should be (e.g., NCEPLIBS) -> (hpc-stack), so no "e.g.".

Reply:
Should I remove NCEPLIBS and NCEPLIBS-external from Table 1.6 Centralized list of documentation? I just added HPC-Stack to the table. However, HPC-Stack has not built its own ReadTheDocs docs yet, so I just linked to the chapter of the SRW docs that pull in most of the HPC-Stack content. Ideally, I would like EMC/HPC-Stack folks to make their own RTD account, and I've filed an issue to that end, but I'm guessing it's not a priority, even though it's a quick thing. The other options (while waiting) are to link to my docs (no guarantee the fork will stay up-to-date though) or link to the HPC-Stack repo and send people fishing for .rst files. I think I prefer just linking to the pertinent SRW chapter, but let me know what you think. I don't think the issue will be resolved by the EPIC/AUS April 1 deadline for this PR, but hopefully it could be by the SRW release.

Reply:
Since we need this documentation for the release, I would go with linking to the relevant SRW App chapter.  After EMC has their own RTD account, it can be changed later.